sdk/include/psdk/rpcproxy.h

```diff
@@ -24,6 +24,10 @@
 #ifndef __WINE_RPCPROXY_H
 #define __WINE_RPCPROXY_H
 
+#ifdef _URLMON_
+#define PROXY_CLSID_IS {0x79EAC9F1,0xBAF9,0x11CE,{0x8C,0x82,0x00,0xAA,0x00,0x4B,0xA9,0x0B}}
```

Comment:
Remove this define, it's not in the official psdk header. At most it would have to be in urlmon's own main header (if putting it in the cmakefiles doesn't work)

Reply:
I tried to put the define in urlmon_main.h (like in 716613cddc8039eff5f095051dc9a79727a25e73), but it seems to never reach the rpcproxy.h header.

Reply:
Would the issue be somehow related to `precomp.h`?

Reply:
The problem is that the define should be in the widl-generated files.